test_infra/ess/deployment.tf

```diff
@@ -73,10 +73,20 @@ locals {
     },
     yamldecode(file("${path.module}/../../pkg/testing/ess/create_deployment_csp_configuration.yaml")))
 
-
-  integration_server_docker_image = coalesce(var.integration_server_docker_image, local.ess_properties.docker.integration_server_image, "docker.elastic.co/cloud-release/elastic-agent-cloud:${var.stack_version}")
-  elasticsearch_docker_image = coalesce(var.elasticsearch_docker_image, local.ess_properties.docker.elasticsearch_image, "docker.elastic.co/cloud-release/elasticsearch-cloud-ess:${var.stack_version}")
-  kibana_docker_image = coalesce(var.kibana_docker_image, local.ess_properties.docker.kibana_image, "docker.elastic.co/cloud-release/kibana-cloud:${var.stack_version}")
+  integration_server_docker_image = coalesce(
```

Comment:
ok why you need to remove the fallback part here? IMO this should be left in, thus, when somebody removes the versions from the yaml it can fallback to the stack_version

Reply:
If someone removes, for example, `docker.integration_server_image` from `create_deployment_csp_configuration.yaml`, the build will expectedly fail. 

> This PR again should and it can target both the CI and the developers, thus, it needs to be based on create_deployment_csp_configuration.yaml, which is already a common point.

Do you want to fall back to `STACK_VERSION`( the latest untested snapshot)?

Reply:
yes this shouldn't fail, this should fallback to `var.stack_version` as it was. Because if in the future for any reason we need to unpin this, we can just remove the entries from `create_deployment_csp_configuration.yaml` and just disable the github action, simple and straight forward without surprises 🙂

Reply:
Ok. Added the fallback to `stack_version`. Also added a comment to the `create_deployment_csp_configuration.yaml` that warns about any big changes